src/Queue/QueueBase.php

```diff
@@ -117,7 +117,7 @@ protected function getQueue(AMQPChannel $channel, array $options = []) {
 
     // Allow modules to override queue settings.
     if (isset($queue_info[$this->name])) {
-      $queue_options += $queue_info[$this->name];
```

Comment:
You're right there is an error, but the fix should not remove defaults as it does when using a plain `=`, but merge new options, using `array_merge()` instead  of `+=`: the latter won't overwrite existing keys, preventing the right member for overriding the default if it is already set. That way it will be possible to overwrite one or more of the default keys without losing them all in the process.